src/app/main.rs

```diff
@@ -495,6 +496,7 @@ where
                 // extension into each request so that all lower metrics
                 // implementations can use the route-specific configuration.
                 let dst_route_stack = phantom_data::layer()
+                    .push(insert_target::layer())
```

Comment:
It's hard to keep straight, so what is the target at this stage?

Reply:
The comment directly above this says: _A per-`dst::Route` layer..._

Would you prefer to explicit type annotations?